param_persist/serialize/serializer.py

```diff
@@ -57,6 +60,59 @@ def to_json(cls, param_object):
         param_json = json.dumps(param_dict)
         return param_json
 
+    @classmethod
+    def from_dict(cls, param_dict):
+        """
+        Create an instance of a param clam from a dictionary.
+
+        Args:
+            param_dict (dict): A dictionary representation of a param class.
+
+        Returns:
+            An instance of a param class defined by the dictionary.
+        """
+        class_path = param_dict.get('class_path')
+
+        if not class_path:
+            raise RuntimeError('Param not configured correctly. Missing "class_path" definition.')
+
+        dict_params = param_dict.get('params')
+
+        if not dict_params:
+            raise RuntimeError('Param not configured correctly. Missing "params" definition.')
+
+        try:
+            class_base_path, class_name = class_path.rsplit('.', 1)
+            param_module = importlib.import_module(class_base_path)
+            param_class = getattr(param_module, class_name)
+        except ImportError:
+            raise RuntimeError(f'Defined param class "class_path" was not importable. Given path is "{class_path}"')
+
+        param_object = param_class()
```

Comment:
Let's use `parameterized_class` instead of `param_class` to more carefully distinguish between Parameterized and Param objects.

Reply:
good point.